ape_starknet/transactions.py

```
@@ -243,6 +243,11 @@ class StarknetReceipt(ReceiptAPI, StarknetBase):
     """Aliased"""
     txn_hash: str = Field(alias="hash")
 
+    @property
+    def return_value(self) -> Any:
+        # Override from ReceiptAPI
+        return None
```

Probably should raise here instead as unsupported